src/components/productsList/_productsList.scss

```diff
@@ -0,0 +1,121 @@
+$media-tablet: 768px;
+$media-desktop: 1024px;
+$padding-mobile: 20px;
+$padding-desktop: 40px;
```

Comment:
For this example's sake this is fine. In an actual project you would move these to a variables file for instance